packages/components/organisms/f-checkout/test-utils/component-objects/f-checkout.component.js

```diff
@@ -169,6 +172,32 @@ module.exports = class Checkout extends Page {
         return this.errorMessageDupOrderGoToHistory.click();
     }
 
+    getCheckoutErrorTabOrder () {
```

Comment:
This one and the one below is quite similar, could they be made into one method instead?

Reply:
Currently this and the method below are serving 2 different expected tab orders. Which is why they are split into 2 methods. We had the decision to either have the expected tab loop order defined in the test or have them defined in the method and found that to be a neater solution due to the fact that multiple tests would be using the same tab loop order.
This is why they are 2 methods.